src/options.rs

```diff
@@ -185,6 +194,23 @@ pub fn builder_from_flags<I>(args: I)
         }
     }
 
+    if let Some(what_to_generate) = matches.value_of("generate") {
```

Comment:
Yeah, I agree with that, in general I want to clean all the options up once we start publishing this crate, so I decided to keep them for now... I thing it's not too unexpected to override with the `--ignore` flags.